Dockerfile

```diff
@@ -149,7 +153,7 @@ WORKDIR /
 
 # We cannot cache this layer otherwise we'll end up with a bigger image
 RUN apt-get update && \
-    apt-get upgrade -y && \
+    apt-get dist-upgrade -y && \
```

Comment:
This is done in [goauthentik/fips](https://github.com/goauthentik/fips) as one of the final steps, so we can save time here.